VISUALIZERS/PLOTLY/BAR/BAR.py

```diff
@@ -7,15 +7,16 @@
 
 @flojoy
 def BAR(default: OrderedPair | DataFrame | Matrix | Vector) -> Plotly:
-    """The BAR node creates a Plotly Bar visualization for a given input data container.
+    """
+    The BAR node creates a Plotly Bar visualization for a given input data container.
 
     Parameters
     ----------
     None
 
-    Supported DC types:
-    ----------------
-    `ordered_pair`, `dataframe` (including timeseries), `matrix`, `vector`
+    Supported DC types
+    ------------------
+    ordered_pair, dataframe (including timeseries), matrix, vector
```

Comment:
"Supported DC types" is not an allowed section, should just be an "Inputs" section
```
Inputs
------
default : OrderedPair|DataFrame|Matrix|Vector
```